content/02.introduction.md

```diff
@@ -6,30 +6,31 @@ Most scientists, including ecologists and evolutionary biologists, depend on com
+Beyond storage and record-tracking, file management in Github is integrated with several communication features, allowing users to engage in discussions, to plan and collaborate in code, and to selectively publish information (Github Issues, Github Discussions, Github Pages; see [Box 1](#definitions)).
+These features contrast with traditional practices of directly sharing files through email, other cloud-based services or physical storage units, which become challenging and time-consuming in long-term and collaborative projects [@doi:10.1186/1751-0473-8-7].
+By allowing users to easily integrate versioning, communication, collaboration with research code and data, Github paves the way for easier implementation of open science practices within research projects [@doi:10.1371/journal.pcbi.1004947].
 
-Git is the version control system that enables all the collaborative tools available on GitHub.
-Git was originally developed as a fast, lightweight and open-source system that would allow software engineers to efficiently develop and collaborate in projects [@url:https://git-scm.com/book/en/v2/Getting-Started-A-Short-History-of-Git]. 
-Since its launch in 2005, Git has become the leading version control system in software development and in other disciplines that similarly require collaboration and community contributions, such as scientific research [@doi:10.1109/MS.2012.61].
-Although the understanding of basic concepts of Git (such as commit, push, pull, checkout; see [Box 1](#definitions)) is necessary, the GitHub web-based platform and its integrated development environments (such as GitHub Desktop) allow users to perform most repository and data management operations without opening Git command-line sessions, making these functionalities available to users who are less familiar with software development.
+Git is the version control system that enables the collaborative tools available on GitHub.
+Git was originally developed as a fast, lightweight and open-source system to allow software engineers to efficiently develop and collaborate in projects [@url:https://git-scm.com/book/en/v2/Getting-Started-A-Short-History-of-Git]. 
+Since its launch in 2005, Git has become the leading version control system in software development and in other disciplines that require collaboration and community contributions, such as in scientific research [@doi:10.1109/MS.2012.61].
+Understanding basic concepts of Git (such as commit, push, pull, checkout; see [Box 1](#definitions)) is recommended to excel in the control of file versions in Github.
+Nevertheless, the GitHub web-based platform and its integrated development environments (such as GitHub Desktop) allow users to perform most repository and data management operations without opening Git command-line sessions, making these functionalities available to users who are less familiar with software development.
```

Comment:
```suggestion
Nevertheless, the GitHub web-based platform and its integrated development environments (such as GitHub Desktop) allow users to perform most repository and data management operations without using the command-line, making these functionalities available to users who are less familiar with software development.
```

Reply:
Modified to "command console", and incorporated.